lib/gems/pending/util/mount/miq_generic_mount_session.rb

```diff
@@ -490,6 +490,10 @@ def settings_mount_point
   end
 
   def source_for_log
-    @input_writer ? "<STREAMED_FROM_CMD>" : @source_input.path
+    case
+    when @input_writer then "<STREAMED_FROM_CMD>"
+    when @source_input then @source_input.path
+    else "<NOT_READY>"
```

Comment:
At the very minimum I would argue that we should be using the instance methods rather than referencing instance variables that don't even exist in this file.

I also feel like there should be an inheritable method that does this instead of defining it in a subclass.

I also prefer `'if` here, as case without an argument just looks odd to me.

Reply:
> At the very minimum I would argue that we should be using the instance methods rather than referencing instance variables that don't even exist in this file.

Fair enough, I think that is a reasonable request.  Granted, these were meant to be private variables, which is why I didn't create methods for them but I think your argument makes sense.

> I also prefer 'if here...

Fine, you twisted my arm... I will do it...

😄

Reply:
@djberg96 So I changed the if, but didn't do the other request.

It isn't that I don't disagree with it, but I want to mostly fix the bug in this PR, and address the major refactoring that this code desperately needs at another time.  This code I know will work, where as I might be introducing other bugs if I try and fix the inheritance and instance variable usage further.

Hope that makes sense.